commands/behold.js

```diff
@@ -0,0 +1,25 @@
+const Augur = require("augurbot"),
+    u = require("../utils/utils");
+
+const Module = new Augur.Module()
+    .addCommand({
+        name: "Behold",
+        description: "creates a doofenshritz behold meme, with the arguments becoming your bottom text. Small chance of fire.",
+        permissions: (msg) => msg.member && msg.channel.permissionsFor(msg.member).has(["ATTACH_FILES", "EMBED_LINKS"]),
```

Comment:
For the moment, let's keep this to #gifs-and-memes.
```suggestion
        permissions: (msg) => msg.channel.id == "121755900313731074" && msg.channel.permissionsFor(msg.member).has(["ATTACH_FILES", "EMBED_LINKS"]),
```